composer.json

```diff
@@ -72,7 +73,8 @@
         "symfony/debug-bundle": "^5.0",
         "symfony/phpunit-bridge": "^5.0",
         "symfony/profiler-pack": "^1.0",
-        "symfony/var-dumper": "^5.0"
+        "symfony/var-dumper": "^5.0",
+        "symfony/web-server-bundle": "4.4"
```

Comment:
> Hmm... are you sure we are not missing some direct dependencies from our composer? Libraries like lcobucci/jwt? We should have them listed here I think

That's a good question: because for now `lcobucci/jwt` is a dependency of `oat-sa/lib-lti1p3-core` which we are dependent from. Since we cannot go with a version higher than the one guys require in lti1p3-core - do you think it would be useful to set version in here, @Skyld ?

Reply:
Yes since multiple features are depending on that library. If we remove the LTI 1.3 bundle one day (theoretically) we don't want to break the JWT authentication.